packages/app/src/app/components/CreateNewSandbox/CreateSandbox/CreateSandbox.tsx

```diff
@@ -15,67 +14,77 @@ import {
   DashboardButton,
 } from './elements';
 import { Explore } from './Explore';
-import {
-  CodeSandboxIcon,
-  PlusIcon,
-  StarIcon,
-  UploadIcon,
-  BackIcon,
-} from './Icons';
+import { BackIcon } from './Icons';
 import { Import } from './Import';
-import { Welcome } from './Welcome';
 
 export const COLUMN_MEDIA_THRESHOLD = 1600;
 
+interface PanelProps {
+  tab: TabStateReturn;
+  id: string;
+  children: ReactNode;
+}
+
+/**
+ * The Panel component handles the conditional rendering of the actual panel content. This is
+ * done with render props as per the Reakit docs.
+ */
+const Panel = ({ tab, id, children }: PanelProps) => {
+  return (
+    <TabContent {...tab} stopId={id}>
+      {({ hidden, ...rest }) =>
+        hidden ? null : (
+          <div hidden={hidden} {...rest}>
+            {children}
+          </div>
+        )
+      }
+    </TabContent>
+  );
+};
+
 interface CreateSandboxProps {
   collectionId?: string;
-  initialTab?: 'Import';
+  initialTab?: 'import';
   isModal?: boolean;
 }
 
-export const CreateSandbox: React.FC<CreateSandboxProps> = props => {
-  const { isFirstVisit, hasLogIn } = useAppState();
+export const CreateSandbox: React.FC<CreateSandboxProps> = ({
+  collectionId,
+  initialTab,
+  isModal,
+}) => {
+  const { hasLogIn } = useAppState();
   const actions = useActions();
 
   const tab = useTabState({
     orientation: 'vertical',
-    selectedId:
-      props.initialTab ||
-      (isFirstVisit && !(window.screen.availWidth < 800)
-        ? 'Welcome'
-        : 'Create'),
+    selectedId: initialTab || 'create',
   });
 
   useEffect(() => {
+    /* ❗️ TODO: We can probably take this out of the useEffect and put it into the useTabState selectedId */
     if (location.pathname.includes('/repositories')) {
       tab.select('Import');
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  const dashboardButtonAttrs = () => {
-    if (location.pathname.includes('/dashboard')) {
-      return {
+  const dashboardButtonAttrs = location.pathname.includes('/dashboard')
+    ? {
+        onClick: actions.modals.newSandboxModal.close,
+      }
+    : {
+        to: '/dashboard',
```

Comment:
probably this one should be /dashboard/home or soon /dashboard/recent, but there's redirect to catch any other route to turn it into home, so not a problem

Reply:
Changed it for clarity 👍